collect_app/src/main/java/org/odk/collect/android/preferences/qr/QRScannerFragment.java

```diff
@@ -83,6 +83,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         Intent intent = new IntentIntegrator(getActivity())
                 .setDesiredBarcodeFormats(getSupportedCodeFormats())
+                .setPrompt(getContext().getString(R.string.barcode_scanner_prompt))
```

Comment:
I think we should make this a separate change for v1.28. Most translators won’t have time to get to this in the next few days so the strings will show up in English where they were translated before.

Reply:
We have over 20 translations for that string the default zxing message also is translatable but I noticed it contains less translations so maybe it's better to have this our own?.

Reply:
Oh, right, this string was already in use for the barcode widget. This change just also uses it in the settings QR code context. That makes a lot of sense, thanks.